### ISSUE TITLE
hinge loss question

### ISSUE THREAD

**Issue by albertofernandezvillan** [mohamadmansourX/SauvolaNet-Training#2]
In the paper, they employ a modified hinge loss function:
`loss[i, j] = max(1 − α · (D[i, j] − T[i, j]) · B[i, j], 0)`

Here, the training procedure uses hinge as loss function:
```
Train:
  optimizer: adam
  loss: hinge
  ....
```

So I understand that the weights included here ([Sauvola_demo.h5](https://github.com/mohamadmansourX/SauvolaNet-Training/blob/master/pretrained_models/Sauvola_demo.h5)) are obtained using hinge loss, while the official pre-trained weights ([Sauvola_v3_att_w7.15.23.31.39.47.55.63_k1_R1_a1_inorm_S256_R0.02_B8_lr0.001_E30-Acc0.9895-Tacc0.9282-F0.9448-PSNR20.59.h5](https://github.com/Leedeng/SauvolaNet/blob/main/pretrained_models/Sauvola_v3_att_w7.15.23.31.39.47.55.63_k1_R1_a1_inorm_S256_R0.02_B8_lr0.001_E30-Acc0.9895-Tacc0.9282-F0.9448-PSNR20.59.h5)) are obtained using modified hinge loss function.

Is this correct?
